draft-ietf-quic-multipath.md

```diff
@@ -250,6 +242,12 @@ the endpoints MUST fallback to
 {{QUIC-TRANSPORT}} with single active path and MUST NOT use any frame or
 mechanism defined in this document.
```

Comment:
Given that we drop support for single packet number space and zero-length CID, we need to change the value of `enable_multipath`. Running a random generator gave me `0x0f739bbc1b666db9`, but we can of course choose another value (for instance, `0x0f739bbc1b666dXX` where `XX` is the draft version; the next version 04 of this document could use  `0x0f739bbc1b666d04`). The IANA table should be updated as well.

Reply:
Yes, I agree that we need to change. Just need to decide with number.

Reply:
Just committed 13e6610576 to address this.